en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-to-soap-api.md

```diff
@@ -0,0 +1,290 @@
+
+The following diagram illustrates the request flow from client to the backend through WSO2 Micro Integrator and the response flow from backend to client through WSO2 Micro Integrator for this example. The WSO2 Micro Integrator acting as the backend for the Choreo Connect Gateway whereas its handling the `JSON` to `SOAP` message transformation as well as `GET` to `POST` method transformation.
+
+<img src="{{base_path}}/assets/img/deploy/mgw/soap-to-rest.png" alt="SOAP to REST request flow" width="650px"/>
+
+| Numbers | Description                                                 |
+|---------|-------------------------------------------------------------|
+| 1       | Client request                                              |
+| 2,3     | Request validation flow via the Enforcer                    |
+| 4       | If the request is valid (i.e. authenticated, not rate-limited etc.), the Router forwards the request to the MI |
+| 5       | After perform the transformation, MI forwards the transformed request to the backend |
+| 6       | Response from the backend |
+| 7       | MI sends transformed response back to the router |
+| 8       | Response to the client |
+
+!!! Note
+    This guide uses [Phone Verify](http://ws.cdyne.com/phoneverify/phoneverify.asmx) from CDYNE as the backend (WSDL: `http://ws.cdyne.com/phoneverify/phoneverify.asmx?wsdl`). You may use your own backend and continue with the same steps.
+
+## Create SOAP to REST transformation using WSO2 Micro Integrator
+
+Following steps will guide you through creation of SOAP to REST transformation using WSO2 Micro Integrator.
+
+!!! info "Before you begin"
+
+    This guide assumes that you already have installed WSO2 Integration Studio, if not you can follow up instructions on [Installing WSO2 Integration Studio]({{base_path}}/integrate/develop/installing-wso2-integration-studio/).
+
+### Step 1 - Create an Integration project with ESB Configs module and Composit Exporter module
+
+Navigate to `File->New->Integration Project` from Integration studio and enter the project name as `PhoneVerification`.
+
+Make sure to tick the **Create ESB Configs** and **Create Composite Exporter** when creating the Integration project.
+
+For more information refer to [Creating an Integration Project]({{base_path}}/integrate/develop/create-integration-project/).
+
+### Step 2 - Create the REST API
+
+1. In the left hand side project explorer panel, expand the created project `PhoneVerification`.
+2. Right click on `PhoneVerfificationConfigs` and select **New->REST API**.
+3. Select the **Import API Artifact** option and provide file having content of [this](link from mgw sample project).
+
+!!! Note
```

Comment:
mention about the sample

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/6114/commits/d03a84262c3c0fa449b465e58f50d312a1220157